### PR TITLE
Add purpose.md

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Add purpose.md to host the Radworks purpose doc

### CODE REVIEW

File: purpose.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+2. Decentralization - That the internet should be decentralized and governed by its users, rather than controlled by governments or corporations.
+3. Security - That technologies should be developed to protect individual privacy and prevent surveillance. 
+4. Accessibility - That technologies and information should be open and accessible to everyone, regardless of financial status or location.
+5. Community - That technology should be developed to promote and harness community, collaboration, and peer-to-peer networks.
+6. Resilience - That technology should be developed to be resistant to censorship, capture, and anti-competitive forces.
+7. Sustainability - That technology should be developed with a long-term perspective, and should prioritize technical and financial sustainability.
+
+## Purpose Statement
+The DAO funds new resilient, permissionless technologies to cultivate internet freedom. We focus on funding and supporting the creation of censorship-resistant and decentralized technologies that empower builders and creators to collaborate. We are committed to enabling individuals and communities to collaborate in the digital realm securely and freely.
```
Comment (shelbeeee) [inline purpose.md]: Add comma between "new" and "resilient"

File: purpose.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Our Purpose 🌱
+
+## Preamble 
+The purpose serves to constitute Radworks, memorializing what we believe and the organization’s raison d’etre. It’s intended as a living document: a North Star that the community can return to for clarity in times of change, challenge, confusion, or conflict. If successful, this document will help protect and further Radwork’s purpose over time as we collectively navigate change.
```
Comment (shelbeeee) [inline purpose.md]: Change It's to Its in 2nd sentence

**Review by shelbeeee — DISMISSED** [review]

File: purpose.md (hunk 2 above)
Comment (abbey-titcomb) [reply purpose.md]: I don't think its would be grammatically correct, the sentence reads as "it is intended..." so I think it's is appropriate.

File: purpose.md (hunk 1 above)
Comment (abbey-titcomb) [inline purpose.md]:
```suggestion
The DAO funds new, resilient, permissionless technologies to cultivate internet freedom. We focus on funding and supporting the creation of censorship-resistant and decentralized technologies that empower builders and creators to collaborate. We are committed to enabling individuals and communities to collaborate in the digital realm securely and freely. 
```

**Review by abbey-titcomb — COMMENTED** [review]
fixed all changes

File: purpose.md (hunk 2 above)
Comment (abbey-titcomb) [inline purpose.md]:
```suggestion
The purpose serves to constitute Radworks, memorializing what we believe and the organization’s raison d’etre. It’s intended as a living document: a North Star that the community can return to for clarity in times of change, challenge, confusion, or conflict. If successful, this document will help protect and further Radworks' purpose over time as we collectively navigate change. 
```

File: purpose.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Our Purpose 🌱
+
+## Preamble 
+The purpose serves to constitute Radworks, memorializing what we believe and the organization’s raison d’etre. It’s intended as a living document: a North Star that the community can return to for clarity in times of change, challenge, confusion, or conflict. If successful, this document will help protect and further Radworks' purpose over time as we collectively navigate change. 
+
+## Foundational Beliefs
+1. Freedom - That technology should be developed to promote freedom and privacy, no matter where you are in the world.
+2. Decentralization - That the internet should be decentralized and governed by its users, rather than controlled by governments or corporations.
+3. Security - That technologies should be developed to protect individual privacy and prevent surveillance.
```
Comment (abbey-titcomb) [inline purpose.md]:
```suggestion
3. Privacy - That technologies should be developed to protect individual privacy and prevent surveillance. 
```

**Review by abbey-titcomb — COMMENTED** [review]
changing security -> privacy re: @cloudhead review

**Review by cloudhead — APPROVED** [review]

**Pull request merged**